ovos_utils/skills/audioservice.py

```diff
@@ -34,7 +35,7 @@ def ensure_uri(s):
             return 'file://' + abspath(s)
         else:
             return s
-    elif isinstance(s, (tuple, list)):
+    elif isinstance(s, (tuple, list)):  # TODO: What case is this handling?
```

Comment:
the api supports passing `(mime, uri)`, let's log a deprecation warning to move over to OCP api directly in this whole class

Reply:
I don't see this implemented there yet, am I missing something?

Reply:
https://github.com/OpenVoiceOS/ovos-core/pull/181

Reply:
https://github.com/OpenVoiceOS/ovos-utils/pull/120